src/app/components/MPulseBeacon/index.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { string } from 'prop-types';
+
+const snippet = apiKey => `
+  window.BOOMR_config = window.BOOMR_config || {};
+  BOOMR_config.autorun = false;
+  BOOMR_config.History = {
+      auto: true,
+      enabled: true
+  };
+
+  (function(){
+    if (window.BOOMR && window.BOOMR.version) { return; }
+    var dom,doc,where,iframe = document.createElement("iframe"),win = window;
+
+    function boomerangSaveLoadTime(e) {
+      win.BOOMR_onload=(e && e.timeStamp) || new Date().getTime();
+    }
+    if (win.addEventListener) {
+      win.addEventListener("load", boomerangSaveLoadTime, false);
+    } else if (win.attachEvent) {
+      win.attachEvent("onload", boomerangSaveLoadTime);
+    }
+
+    iframe.src = "javascript:void(0)";
+    iframe.title = ""; iframe.role = "presentation";
+    (iframe.frameElement || iframe).style.cssText = "width:0;height:0;border:0;display:none;";
+    where = document.getElementsByTagName("script")[0];
+    where.parentNode.insertBefore(iframe, where);
+
+    try {
+      doc = iframe.contentWindow.document;
+    } catch(e) {
+      dom = document.domain;
+      iframe.src="javascript:var d=document.open();d.domain='"+dom+"';void(0);";
+      doc = iframe.contentWindow.document;
+    }
+    doc.open()._l = function() {
+      var js = this.createElement("script");
+      if (dom) { this.domain = dom; }
+      js.id = "boomr-if-as";
+      js.src = "https://s.go-mpulse.net/boomerang/${apiKey}";
```

Comment:
When running this locally it throws a console error which isnt a great dev experience. Can you make a dev route for this, like we do for the cookie oven https://github.com/bbc/simorgh/blob/latest/src/server/index.jsx#L106-L110. Idm how its passed in after that, for the cookie oven we did https://github.com/bbc/simorgh/blob/24dc348c7f7235c2fea505a5404882f0db2fd273/src/app/containers/ConsentBanner/CanonicalLogic/cookieOvenUrl.js but it could also be simorgh env variables

Reply:
The API key is currently declared in an env var https://github.com/bbc/simorgh/blob/0faf1cea20aa375292bf38b8eb384b77c2fac545/envConfig/local.env#L5

Either removing this var or entering the correct API key would resolve the console error. Happy to remove the default but we would need to review the testing strategy so that we can continue to test against the beacon in e2es, lighthouse etc.